### ISSUE TITLE
Introduce common contrib

### ISSUE THREAD

**Issue by ben-hamelin** [oomphinc/drupal-scaffold#95]
@syzygy333 per our meeting the other day, here are some common modules I think we can consider:

- pathauto
- redirect
- metatag
- admin_toolbar
- crop API
- Image Widget Crop
- token
- webform
- upgrade status
- XML sitemap

Are there other modules or projects you had in mind?

**Comment by syzygy333** [comment]
Beautiful. I love these choices.

Some possible additions (all up for discussion, though I end up using these on 95% of sites):

- better_exposed_filters
- coffee
- config_split
- permissions_filter
- rabbit_hole
- simple_sitemap
- smtp

**Comment by ben-hamelin** [comment]
How does the Drupal 10 recipes initiative impact this approach?

**Comment by syzygy333** [comment]
Yeah, if recipes shakes out like they are saying, we could easily have a "base" recipe that contains these and they could even be broken up into "base | admin," "base | media," "base | content" etc and people can just choose to use those recipes or not. I guess the question is "is there value in adding these now as we wait for recipes to land in a real way?"
